### PR TITLE
Person features

### PR DESCRIPTION
**Pull request by lucasalencar**
Based on the movie's features, I made some basic API calls.

Method #find that gets the basic information about a person given a specific ID.

``` ruby
person = TMDb::Person.find(138)
```

Method #images that gets the images for a specific person ID.

``` ruby
images = TMDb::Person.images(138)
```

Method #popular that returns a list of popular people from TMDb. Accepts page option.

``` ruby
popular = TMDb::Person.popular
popular = TMDb::Person.popular(page: 4)
```

### CODE REVIEW

File: spec/fixtures/person/images.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	"profiles": [
+		{
+			"file_path": "/j77Z3f2m0e211ocFhPJu5ZiO12R.jpg",
+			"width": 290,
+			"height": 290,
+			"iso_639_1": null,
+			"aspect_ratio": 1.0
+		}
+	]
+}
```
Comment (andrielfn) [inline spec/fixtures/person/images.json]: Try to fix this putting a blank line at the end of the file.

**Comment by andrielfn** [comment]
I changed a bit how we are organizing the specs, take a look here https://github.com/andrielfn/tmdb-api/commit/fd21b2838d00341dcad6aaf3218a8da0239ab1eb

File: spec/tmdb-api/person_spec.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      popular = TMDb::Person.popular
+
+      expect(popular.first.adult).to eq(false)
+      expect(popular.first.name).to eq('Daniel Craig')
+      expect(popular.first.profile_path).to eq("/wWMdqiqW6unT6TnXydQbOtYffeO.jpg")
+    end
+
+    it "raises with a bad request" do
+      stub_get('/person/popular').with(query: { page: 200 }).to_return(status: 404)
+      expect{ TMDb::Person.popular(page: 200).to raise_error ArgumentError }
```
Comment (andrielfn) [inline spec/tmdb-api/person_spec.rb:68]: Change this line to `expect{ TMDb::Person.popular(page: 200) }.to raise_error ArgumentError`

File: spec/tmdb-api/person_spec.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  describe '.popular' do
+    it "gets a list of popular people" do
+      stub_get('/person/popular').to_return(json_response('person/popular.json'))
+
+      popular = TMDb::Person.popular
+
+      expect(popular.first.adult).to eq(false)
+      expect(popular.first.name).to eq('Daniel Craig')
+      expect(popular.first.profile_path).to eq("/wWMdqiqW6unT6TnXydQbOtYffeO.jpg")
```
Comment (andrielfn) [inline spec/tmdb-api/person_spec.rb:63]: Try to use always single quotes.

**Comment by andrielfn** [comment]
Don't forget to add this new features in the README.

File: README.md [hunk 4]
```diff
@@ -192,6 +192,81 @@ movie.vote_average # => 8.2
...
+person.biography # => "From Wikipedia, the free encyclopedia\n\nQuentin Jerome Tarantino is an American film director, screenwriter, producer, cinematographer and actor. ..."
+person.birthday # => "1963-03-27"
+person.deathday # => "Amém"
+person.homepage # => "http://www.tarantino.info/"
+person.imdb_id # => "nm0000233"
+person.name # => "Quentin Tarantino"
+person.place_of_birth # => "Knoxville, Tennessee, USA"
+person.popularity # => 5.26352072707883
+person.profile_path # => "/iws3gBdQJ6tF7x6CIpepYfUKc58.jpg"
+```
```
Comment (andrielfn) [inline README.md:247]: Remove this "object" section and shows the availables attributes, as I did here https://github.com/andrielfn/tmdb-api/commit/2fe9eb1e917b7035c6550dd0b6937bd195170709

**Comment by fernandomantoan** [comment]
Nice, i think that the standard of using single or double quotes could give more consistency to the code, and can help people who want to contribute and don't know if they should use ' or ". What do you think about having a way to call a `next()` in the `TMDb::Person.popular`? Although it is easy to calculate the next page, it could be handy somewhat for the developers.

File: README.md [hunk 5]
```diff
@@ -171,6 +171,81 @@ TMDb::Movie.upcoming
 #    ]
 ```
 
+### Find a person by ID
+
+Get the basic person information for a specific person id.
+
+```ruby
+TMDB::Movie.find(138)
```
Comment (andrielfn) [inline README.md:179]: Change to `TMDb::Person.find(138)`

File: lib/tmdb-api.rb [hunk 6]
```diff
@@ -12,9 +12,14 @@
 require 'tmdb-api/production_country'
 require 'tmdb-api/spoken_language'
 require 'tmdb-api/changes'
+require 'tmdb-api/person'
 
 require 'tmdb-api/version'
 
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.uncountable %w( also_known_as )
+end
```
Comment (andrielfn) [inline lib/tmdb-api.rb:21]: Move this config to a `inflections.rb` file os something like that.

File: lib/tmdb-api/base.rb [hunk 7]
```diff
@@ -38,11 +38,24 @@ def initialize(attributes = {})
...
+    #
+    # value - value to be evaluated
+    #
+    # Returns true or false
+    def candidate_to_object?(value)
+      if value.is_a?(Array) and value.size > 0
+        !value.first.is_a? String
+      else
+        false
+      end
```
Comment (andrielfn) [inline lib/tmdb-api/base.rb:56]: I think you can have something like this:

``` ruby
def candidate_to_object?(value)
  value.is_a?(Array) && (value.map.select { |h| h.is_a?(Hash) } == value)
end
```

So we ensure that only arrays of hashes could be objects.

**Pull request merged**